### ISSUE TITLE
Should I be able to open saved studio states from a different minor version?

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/gravity-and-orbits#476]
For https://github.com/phetsims/qa/issues/850, things looks good when I open a saved studio state from 1.5 into 1.6.0-rc.4. When I tried to upload a saved state from 1.6.0-rc.1, I got an 'incorrect username or password' message. Should I be able to load older minor versions into the latest version?
<img width="519" alt="Screenshot 2022-11-11 at 1 49 17 PM" src="https://user-images.githubusercontent.com/87318828/201413065-5e24a4de-050d-4414-93cd-dd5101ddfa74.png">

<img width="504" alt="Screenshot 2022-11-11 at 2 10 10 PM" src="https://user-images.githubusercontent.com/87318828/201413361-6accb9e3-5dc9-4f73-9cba-3ebc8cc9e2f0.png">

**Comment by samreid** [comment]
The problem is that it expects the previous version to be loaded from a production version, hence it is looking in phet-io.colorado.edu/sims/gravity-and-orbits (not on phet-dev).  So I think this will work OK once things are published, and no work is needed for this publication.

To correct this for future versions, each save file could also output the URL of the corresponding API file.  @zepumph does that sound like a good idea to you?

**Comment by zepumph** [comment]
This seems like a real, and important problem to try to solve. Should we promote it to a common issue, perhaps in our backlog?

**Comment by samreid** [comment]
Yes, but I don't know whether to do that by marking it for the "proposed dev activity" column, or by cross-listing it in the "phet-io backlog" issue. And I don't know how it compares to other priority.  Maybe bundle it with an upcoming phet-io sim that needs migration rules?
